FWCore/Framework/src/EventProcessor.cc

```diff
@@ -1275,15 +1272,10 @@ namespace edm {
 
           EventSetupImpl const& es = status->eventSetupImpl(esp_->subProcessIndex());
           {
+            LumiTransitionInfo transitionInfo(lumiPrincipal, es, &status->eventSetupImpls());
```

Comment:
I like how the TransitionInfos make the global work more uniform. I'd suggest we also do the same for the Stream functions.

Reply:
Do you want this in this PR? Or as a separate PR? ProcessBlock does not even use the stream transitions.